layouts/default.vue

```diff
@@ -91,12 +112,26 @@
         set (value) {
           this.updateDrawer(value)
         }
+      },
+      search: {
+        get () {
+          return this.$store.state.searchInput
+        },
+        set (value) {
+          this.updateSearchInput(value)
+        }
+      },
+      logoIsShowing () {
+        if (this.$route.path !== '/listguests') return true
+        else if (!this.$vuetify.breakpoint.xs) return true
```

Comment:
can you change to `!this.$vuetify.breakpoint.xs` to [`this.$vuetify.breakpoint.smAndUp`](https://vuetifyjs.com/en/customization/breakpoints)